Mobile/ShopLens/Droid/Activities/MainActivity.cs

```diff
@@ -87,14 +89,14 @@ protected override void OnCreate(Bundle savedInstanceState)
 
             if (!talkBackEnabled)
             {
-               InitiateNoTalkBackMode();
+                InitiateNoTalkBackMode();
```

Comment:
Not sure if fixing bad indentation...
![image](https://user-images.githubusercontent.com/21359162/49605933-997a2980-f99a-11e8-9bc1-0a0b95bb774c.png)
... or adding bad indentation

Reply:
I swear to god, I didn't do it on purpose.

Reply:
It was Ctrl + K + D, from now on I won't select the whole document text with it.

Reply:
I mean the other formatting changes were made by it.